shi

```diff
@@ -24,6 +24,12 @@ if ! [ -d "$VIRTUAL_ENV" ]; then
   else
     echo "File requirements.txt not found, not installing requirements"
   fi
+else
+  . "$VIRTUAL_ENV/bin/activate"
+  if [[ -f "$SCRIPT_DIR/requirements.txt" ]] ; then
+    echo "Checking for newly added requirements"
+    pip3 install -r requirements.txt
```

Comment:
shouldn't be here "$SCRIPT_DIR/requirements.txt" ?

Reply:
Cool, thanks 👍